containers/jml-extract-lambda/CHANGELOG.md

```diff
@@ -9,11 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [1.3.3] - 2024-03-07
+## [1.4.0] - 2024-03-08
```

Comment:
I think you want to leave the notes for 1.3.3, and create a new section for the new release. See my note below about making this a patch release rather than minor - I always refer to the [semver guidance ](https://semver.org/#summary)when deciding on the version to bump

Reply:
Thanks @michaeljcollinsuk i didnt spot that i had overwritten the note will amend